### DISCUSSION TITLE
PHP

### DISCUSSION THREAD

**Discussion by raxil2024** [recodehive/Stackoverflow-Analysis#448] [Q&A] [answered]
What are your views on PHP.. espcially its framework laravel livewire??

**Comment by SuBroXz11** [comment]


**Reply by raxil2024** [reply, marked as answer]
thanks i am learnng laravel right now

**Reply by Prince-Tirkey** [reply]
happy learning🌟

**Comment by Prince-Tirkey** [comment]
Php is still a good language with huge community. Laravel is definitely more famous and widely used compared to Livewire.

**Comment by gbad8** [comment]
Well, I love PHP.
PHP is easy to learn and very intuitive. It powers 70% of websites and has a large, active community.

Are you thinking about learning it? If so, I encourage you! Check out this site:[https://phptherightway.com/](url)

**Comment by Michael9621** [comment]
Laravel is PHP framework that makes aids developers build robust applications. It does this by providing elegant syntax,  built in authentication, routing, caching and sessions as well an ORM system called eloquent among others.

Livewire is a full stack framework for Laravel, it allows you to build reactive UIs using only PHP without JavaScript. It uses Ajax behind the scenes to update parts of the UI.  You might also want to check out inertia.js
